src/plugins/vis_type_tagcloud/public/components/tag_cloud.test.js

```diff
@@ -149,7 +152,11 @@ describe('tag cloud tests', () => {
         tagCloud = new TagCloud(domNode, colorScale);
         tagCloud.setData(currentTest.data);
         tagCloud.setOptions(currentTest.options);
-        await fromNode((cb) => tagCloud.once('renderComplete', cb));
+        await new Promise((resolve) => {
+          tagCloud.once('renderComplete', () => {
+            resolve();
+          });
+        });
```

Comment:
Since this piece is used so many times, I would recommend you pull it out into a function of its own; something like:
```js
const waitForRenderCompletion = async (tagCloud) => 
  new Promise((resolve) => {
    tagCloud.once('renderComplete', resolve);
  });
```

Reply:
+1 to both put this func out and use
`tagCloud.once('renderComplete', resolve));` cuz `(() => { resolve(); })` doesn't add anything in terms of functionality; it just calls resolve with no arguments.